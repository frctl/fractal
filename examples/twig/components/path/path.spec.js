const fractal = require('../../fractal.config.js');

describe('path', () => {
    beforeEach(async () => {
        await fractal.load();
    });

    it('renders original path for server', async () => {
        const render = await fractal.components.find('@path').render(undefined, { server: true });
        expect(render).toBe('/some-path\n');
    });

    it('converts absolute path to relative for builder', async () => {
        const render = await fractal.components.find('@path').render(undefined, { builder: true });
        expect(render).toBe('some-path.html\n');
    });
});
