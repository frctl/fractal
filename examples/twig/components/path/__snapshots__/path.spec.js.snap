// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`path converts absolute path to relative for builder 1`] = `
"some-path.html
"
`;

exports[`path renders original path for server 1`] = `
"/some-path
"
`;
